qpc/messages.py

```diff
@@ -209,6 +209,10 @@
 REPORT_NO_DETAIL_REPORT_FOR_REPORT_ID = "No report detail available for report id %s."
 REPORT_NO_INSIGHTS_REPORT_FOR_SJ = "No Insights report available for scan job %s."
 REPORT_NO_INSIGHTS_REPORT_FOR_REPORT_ID = "Insights report %s does not exist."
+REPORT_NO_INSIGHTS_CLARIFICATION = (
+    "Insights report can be generated only if at least one source type "
+    "is network, satellite or vcenter."
```

Comment:
Very minor nitpick. (Oxford comma)
```suggestion
    "is network, satellite, or vcenter."
```